pymc/model_graph.py

```diff
@@ -219,13 +238,18 @@ def make_graph(self, formatting: str = "plain"):
                 for var_name in var_names:
                     self._make_node(var_name, graph, formatting=formatting)
 
-        for key, values in self.make_compute_graph().items():
-            for value in values:
-                graph.edge(value.replace(":", "&"), key.replace(":", "&"))
+        for child, parents in self.make_compute_graph(selected_vars=selected_vars).items():
+            # parents is a set of rv names that preceed child rv nodes
+            for parent in parents:
+                try:
+                    graph.edge(parent.replace(":", "&"), child.replace(":", "&"))
+                except AttributeError:
+                    print(child, parent)
```

Comment:
Why print? Can this be a logging statement?

Reply:
Oops, this was left over from when I was debugging 😅 It shouldn't be there